SimG4CMS/Forward/src/FastTimerSD.cc

```diff
@@ -290,43 +277,37 @@ void FastTimerSD::CreateNewHit() {
   StoreHit(currentHit);
 }	 
  
-
 void FastTimerSD::UpdateHit() {
 
-  if (Eloss > 0.) {
-    currentHit->addEnergyDeposit(edepositEM,edepositHAD);
+  currentHit->addEnergyDeposit(edepositEM,edepositHAD);
 
 #ifdef EDM_ML_DEBUG
-    std::cout << "updateHit: add eloss " << Eloss <<std::endl;
-    std::cout << "CurrentHit="<< currentHit<< ", PostStepPoint = "
-	      << postStepPoint->GetPosition() << std::endl;
+  edm::LogVerbatim("FastTimerSD") 
+    << "updateHit: add eloss " << edeposit
+    << "\n CurrentHit="<< currentHit<< ", PostStepPoint = "
+    << postStepPoint->GetPosition();
 #endif
-    currentHit->setEnergyLoss(Eloss);
-  }  
 
   // buffer for next steps:
   tsID           = tSliceID;
   primID         = primaryID;
   previousUnitID = unitID;
 }
 
-
 G4ThreeVector FastTimerSD::SetToLocal(const G4ThreeVector& global){
```

Comment:
should method not start with lowercase?

Reply:
Traying to fix but the rule was broken in many places.